### PR TITLE
Update README.md.

### PR DESCRIPTION
**Pull request by sergeypalkin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
-# Payone Module
+# PayoneModule
```
Comment (dereuromark) [inline README.md]: missing space!

Comment (sergeypalkin) [reply README.md]: @dereuromark Do we have an article with described requirements related to a module description text?

Comment (dereuromark) [reply README.md]: See existing module generator in Development.

ModuleName Module

as for all other 5xx modules.

Comment (sergeypalkin) [reply README.md]: Thanks.

Comment (sergeypalkin) [reply README.md]: Fixed.

**Pull request merged**